core/server/data/migrations/versions/3.18/01-add-email-preview-permissions-to-roles.js

```diff
@@ -0,0 +1,41 @@
+const utils = require('../../../schema/fixtures/utils');
+const {logging} = require('../../../../lib/common');
```

Comment:
logging is now in `core/shared/logging` :eyes:

Reply:
Thanks, good catch, rebased master and updated the path 🙌 Wondering though if there is any way this should have or can in future trigger failure anywhere in tests, as could have been easily missed if all green 🤔

Reply:
Was just thinking the same. @vikaspotluri123 flagged up https://github.com/benmosher/eslint-plugin-import, so I'll look into it.